data_source_obmcs_core_instance.go

```diff
@@ -161,3 +168,18 @@ func (s *InstanceDatasourceCrud) SetData() {
 	}
 	return
 }
+
```

Comment:
it is necessary because in the schema above we had to specify the extended_metadata as a map[string]string
This is because afaik there is no way to specify a nested map of arbitrary depth in the schema